src/javascripts/Views/recordView.js

```diff
@@ -157,11 +156,17 @@ ngapp.controller('recordViewController', function($scope, $element, $timeout, ht
             $scope.buildColumns();
             $scope.buildTree();
             $scope.$broadcast('recordChanged');
+            let referencedByView = $scope.view.linkedReferencedByView;
```

Comment:
I would opt to make this a separate function, to be honest.  Something like `syncReferencedByView`.